### PR TITLE
feat: add prevent-no-label-execution action

### PR DESCRIPTION
**Pull request by kenji-miyake**
For https://github.com/autowarefoundation/autoware.universe/pull/173

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@KeisukeShima Although I'll merge this now for `autoware.universe`, please leave your reviews and questions on this PR if there are any comments. :pray:

File: README.md [hunk 1]
```diff
@@ -35,6 +35,27 @@ jobs:
...
+```yaml
+jobs:
+  prevent-no-label-execution:
+    uses: autowarefoundation/autoware-github-actions/prevent-no-label-execution.yaml@tier4/proposal
+    with:
+      label: ARM64
+
+  build-and-test-arm:
+    needs: check-run-condition
+    if: ${{ needs.check-run-condition.outputs.run == 'true' }}
```
Comment (KeisukeShima) [inline README.md:54]:
```suggestion
    needs: prevent-no-label-execution
    if: ${{ needs.prevent-no-label-execution.outputs.run == 'true' }}
```

This is a minor suggestion.

**Review by KeisukeShima — COMMENTED** [review]
I added a comment. Other parts are LGTM.

File: README.md (hunk 1 above)
Comment (kenji-miyake) [reply README.md:54]: Oh, it was old. Thank you!

Comment (kenji-miyake) [reply README.md:54]: 0e90ce1e8f79e2840d3c3d74d073694b918e2abb

**Pull request merged**